add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
+
+if ($_SERVER['REQUEST_METHOD'] == 'POST') {
+    $title = $_POST['title'];
+    $author = $_POST['author'];
```

Comment:
Observations: Error handling is done by echoing database errors directly to the user, which is not user-friendly and can expose sensitive information.
Recommendations: Implement a logging mechanism to log errors and display a generic error message to the user.

```php
if ($stmt->execute()) {
    echo "Book added successfully!";
} else {
    error_log('Database error: ' . $stmt->error);
    echo "An error occurred while adding the book.";
}
```